### PR TITLE
Added React Native bare example

### PR DESCRIPTION
**Pull request by JDLegaspi**
Nothing special here, I just created an app the way the [RN team recommend](https://reactnative.dev/docs/getting-started-without-a-framework#step-1-creating-a-new-application). I then installed `@thanks/react-native-webview`.

I've installed `react-native-webview` as recommended in the official [Thanks RN README](https://github.com/thanksad/rocket/tree/main/workspace/products/API/react-native), and ran `pod install` inside `ios/`.

I can confirm that both Direct and Indirect apps work in this example.

### CODE REVIEW

File: react-native/src/App.tsx [hunk 1]
```diff
@@ -48,14 +62,26 @@ export default function App() {
...
+    >
+      <ThanksBlock
+        slot="slot-mobile"
+        partnerId="949ca5e1-8a92-4198-948c-a9400fdf9fb1"
+        widgetConfiguration={{
+          animationType: "slide",
+        }}
+        height={152}
+        width={"100%"}
+      />
```
Comment (theKashey) [inline react-native/src/App.tsx]: Let's set height to `SafeArea` and use block that is already available.

File: react-native-bare/App.tsx
Comment (theKashey) [inline react-native-bare/App.tsx:1]: can you please move it to a subfolder.
The root directory is too cluttered and you need to put "the actual code" aside.

Comment (JDLegaspi) [reply react-native-bare/App.tsx:1]: Can do 🫡

File: react-native/src/App.tsx (hunk 1 above)
Comment (JDLegaspi) [reply react-native/src/App.tsx]: `SafeArea` is just a wrapper to all the elements, so I've added a wrapper to the Block itself to set its height.

File: react-native-bare/__tests__/App.test.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import App from '../src/App';
+
+// Note: import explicitly to use the types shipped with jest.
+import {it} from '@jest/globals';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  renderer.create(<App />);
```
Comment (theKashey) [inline react-native-bare/__tests__/App.test.tsx:16]: So we can create an "integration" test for RN

**Review by theKashey — APPROVED** [review]

**Pull request merged**